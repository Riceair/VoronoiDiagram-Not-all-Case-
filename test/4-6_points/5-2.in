5
123 456
456 123
1 1
120 459
400 100