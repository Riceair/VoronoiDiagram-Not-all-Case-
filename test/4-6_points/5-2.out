P 1 1
P 120 459
P 123 456
P 400 100
P 456 123
E -288611.7 75234.34 -71.66 264.34
E -71.66 264.34 163.05 201.41
E -71.66 264.34 1818.34 2154.34
E 163.05 201.41 335.85 335.86
E 163.05 201.41 62570.5 -251319.5
E 335.85 335.86 14918 -35168.5
E 335.85 335.86 210079.5 210079.5
